crates/ide_completion/src/completions/unqualified_path.rs

```diff
@@ -45,6 +47,22 @@ pub(crate) fn complete_unqualified_path(acc: &mut Completions, ctx: &CompletionC
             cov_mark::hit!(skip_lifetime_completion);
             return;
         }
+        if let ScopeDef::Local(local) = &res {
```

Comment:
this isn't really an "unqualified path" we're completing, so I'd think it'd make more sense to separate it?